locale/de.json

```diff
@@ -23,7 +23,7 @@
     "primaryMac": "Primäre MAC",
     "id": "Knoten ID",
     "firstSeen": "Erstmals gesehen",
-    "systemLoad": "Load average",
+    "systemLoad": "Systemlast",
```

Comment:
We use a translation platform - not via PR. This way new languages easy to add and maintain.
https://crowdin.com/project/meshviewer

Reply:
This site apparently needs the user to create an account. That might discourage occasional editors.

Reply:
We won't change that.
- This allows user without git to edit it. Don't worry about breaking characters like `' or "`.
- It easier to keep track of new or changed terms.
- Commit guidelines are not important like Message
- Suggestions from different translation platform are not available for checking to avoid mistakes.

Reply:
github.com allows to edit files and to create merge requests without any git or console knowledge.
A for the text, people (not knowing English) will understand "Systemlast", but not "Load average".
Imho, the advantage of using crowdin.com is rather slim.

Anyway, just my 5cents. :-)

Reply:
Creating a new language isn't possible
List changed terms in original language
And most PR via GitHub are horrible and need extra work.